src/app/adf-roles/adf-roles-routing.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { DfManageRolesComponent } from './df-manage-roles/df-manage-roles.component';
+import { DfCreateRoleComponent } from './df-create-role/df-create-role.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: DfManageRolesComponent,
```

Comment:
FYI ℹ️ We decided to use a resolver to grab the module data on route load

Reply:
Ah, right - thanks!